frontend/lib/resolution/resolution-queries.cpp

```diff
@@ -3625,6 +3625,29 @@ doIsCandidateApplicableInitial(ResolutionContext* rc,
     }
   }
 
+  // For now, ignore any candidates that are not part of the 2.0 edition
+  // Going forward, we'll need to teach Dyno's resolver to respect the
+  // --edition flag.
+  //
+  // Developed by @benharsh
```

Comment:
😄 I don't know if you have to credit @benharsh but I'm sure he'll appreciate it.

Reply:
He handed me the code, so it felt only fair.  (and helps wash my hands of being accused of inserting potential mistakes into Dyno! :D ).